src/server/rdb_test.cc

```diff
@@ -316,4 +316,22 @@ TEST_F(RdbTest, HMapBugs) {
   EXPECT_EQ(2, CheckedInt({"hlen", "hmap1"}));
 }
 
+TEST_F(RdbTest, JsonTest) {
+  string_view data[] = {
+      R"({"a":1})"sv,                          //
+      R"([1,2,3,4,5,6])"sv,                    //
+      R"({"a":1.0,"b":[1,2],"c":"value"})"sv,  //
+      R"({"a":{"a":{"a":{"a":1}}}})"sv         //
+  };
```

Comment:
The comments are there so that the formatter does not stick them into one line 🤔 Or better use `// clang-format on`, but then I have to align them manually 😒